src/components/common/dashboard_contents/pie-chart.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { PieChart } from 'react-minimal-pie-chart';
+import './pie-chart.css';
+import MOCK_DATA from '../../../tempData/trxList.json';
+import PieChartParser from './pie-chart_parser';
+
+const [finalResults, totalSum] = PieChartParser(MOCK_DATA);
+
+// export default function SpendingRender() {
+// const myData = [
+//     finalResults.Bills,
+//     finalResults.Entertainment,
+//     finalResults.Other,
+//     finalResults.CreditCard,
+//     finalResults.FastFood,
+//     finalResults.Income,
+//   ];
+// }
+
+export default function SpendingRender() {
+  const myData = [
+    finalResults.Bills,
+    finalResults.Entertainment,
+    finalResults.Other,
+    finalResults.CreditCard,
+    finalResults.FastFood,
+    finalResults.Income,
+  ];
+
+  return (
+    <div className="chart">
+      <PieChart
+        // your data
+        data={myData}
+        // width and height of the view box
+        viewBoxSize={[100, 100]}
+        startAngle={270}
+      />
+      <div className="details">
+        <ul className="colorReference_red">
+          {myData[0].category}: ${myData[0].value}

```

Comment:
This works, but makes the website less semantic. Giving future users, primarily those with disabilities, a hard time when navigating the site. Is there a way to loop through `myData` and create a list item?